File: mobile/src/main/java/com/windscribe/mobile/help/HelpView.java
/*
 * Copyright (c) 2021 Windscribe Limited.
 */

package com.windscribe.mobile.help;

public interface HelpView {

    void goToSendTicket();

    void openInBrowser(String url);

    void setActivityTitle(String title);

    void showProgress(boolean inProgress, boolean success);

    void showToast(String message);
}

File: mobile/src/main/java/com/windscribe/mobile/help/HelpView.kt
/*
 * Copyright (c) 2021 Windscribe Limited.
 */
package com.windscribe.mobile.help

interface HelpView {
    fun goToSendTicket()
    fun openInBrowser(url: String)
    fun setActivityTitle(title: String)
    fun showProgress(inProgress: Boolean, success: Boolean)
    fun showToast(message: String)
    fun setSendTicketVisibility(visible: Boolean)
}